Gemfile

```diff
@@ -11,56 +11,48 @@ gem 'pg', '~> 1.1'
 gem 'puma', '~> 5.0'
 # Use SCSS for stylesheets
 gem 'sass-rails', '>= 6'
-gem 'bootstrap-sass', '~> 3.4.1'
-gem 'jquery-rails'
 # Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
 gem 'webpacker', '~> 5.0'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.7'
-# Use Redis adapter to run Action Cable in production
-# gem 'redis', '~> 4.0'
-# Use Active Model has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
 gem 'circleci', require: false
 # Devise is a flexible authentication solution for Rails based on Warden
-gem 'devise'
+gem 'devise', '~> 4.7.3'
 # Standardized Multi-Provider Authentication
-gem 'omniauth-github', github: 'intridea/omniauth-github'
-gem 'omniauth-openid', github: 'intridea/omniauth-openid'
-# OmniAuth Facebook
-gem 'omniauth-facebook'
 # Rails engine that provides an easy-to-use interface for managing your data.
-gem 'activeadmin'
 # Provides a set of helpers which guide you in leveraging regular Ruby classes 
-gem "pundit"
+gem "pundit", '~> 2.1.0'
 # Pagy is the ultimate pagination gem
 gem 'pagy', '~> 3.5'
 # Shrine is a toolkit for handling file attachments in Ruby applications.
 gem "shrine", "~> 3.0"
 # Ruby state machines
-gem 'aasm'
+gem 'aasm', '~> 5.1.1'
 # Using MiniMagick the ruby processes memory remains small 
 gem "mini_magick"
 # Fast and powerful Git hooks manager for Node.js, Ruby or any other type of projects.
 gem 'lefthook', '~> 0.5.5'
-
-
-
-
-
-# Use Active Storage variant
-# gem 'image_processing', '~> 1.2'
-
+gem 'mimemagic', '~> 0.4.3'
+gem 'jwt', '~> 2.2.2'
+gem 'haml-rails', '~> 2.0.1'
+gem 'cancancan', '~> 3.2.1'
+gem 'image_processing', '~> 1.2'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
+group :production, :development, :test do
```

Comment:
If the gem is outside from group it means that this gem will used on each environment